sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -116,94 +118,113 @@ def _convert_span_to_envelope(span: Span) -> TelemetryItem:
+                scheme = span.attributes[SpanAttributes.HTTP_SCHEME]
+                http_target = span.attributes[SpanAttributes.HTTP_TARGET]
+                if SpanAttributes.HTTP_HOST in span.attributes:
                     url = "{}://{}{}".format(
                         scheme,
-                        span.attributes["http.host"],
+                        span.attributes[SpanAttributes.HTTP_HOST],
                         http_target,
                     )
-                elif "net.host.port" in span.attributes:
-                    host_port = span.attributes["net.host.port"]
-                    if "http.server_name" in span.attributes:
-                        server_name = span.attributes["http.server_name"]
+                elif SpanAttributes.NET_HOST_PORT in span.attributes:
+                    host_port = span.attributes[SpanAttributes.NET_HOST_PORT]
+                    if SpanAttributes.HTTP_SERVER_NAME in span.attributes:
+                        server_name = span.attributes[SpanAttributes.HTTP_SERVER_NAME]
                         url = "{}://{}:{}{}".format(
                             scheme,
                             server_name,
                             host_port,
                             http_target,
                         )
-                    elif "net.host.name" in span.attributes:
-                        host_name = span.attributes["net.host.name"]
+                    elif SpanAttributes.NET_HOST_NAME in span.attributes:
+                        host_name = span.attributes[SpanAttributes.NET_HOST_NAME]
                         url = "{}://{}:{}{}".format(
                             scheme,
                             host_name,
                             host_port,
                             http_target,
                         )
-            if url:
-                url = url[:2048]  # Breeze max length
             data.url = url
-            data.properties["request.url"] = url
-            if "http.status_code" in span.attributes:
-                status_code = span.attributes["http.status_code"]
+            # Http specific logic for ai.operation.name
+            if SpanAttributes.HTTP_ROUTE in span.attributes:
+                envelope.tags["ai.operation.name"] = "{} {}".format(
+                    span.attributes[SpanAttributes.HTTP_METHOD],
+                    span.attributes[SpanAttributes.HTTP_ROUTE],
+                )
+            elif url:
+                try:
+                    parse_url = urlparse(url)
+                    path = parse_url.path
+                    if not path:
+                        path = "/"
+                    envelope.tags["ai.operation.name"] = "{} {}".format(
+                        span.attributes[SpanAttributes.HTTP_METHOD],
+                        path,
+                    )
+                except Exception:  # pylint: disable=broad-except
+                    logger.warning("Error while parsing url.")
```

Comment:
Maybe just log it?, SpanAttributes could have wrong data not sure we want to report it as warning, I guess it depends what is the difference between log and warning in this exporter

Reply:
Good point. The alternative is just to `pass` on exception (which does nothing). I'll change it to that so the user does not get bombarded with logs.